bpf/lib/lxc.h

```diff
@@ -203,3 +202,4 @@ static inline bool __inline__ tc_index_skip_proxy(struct __sk_buff *skb)
 
 	return tc_index & TC_INDEX_F_SKIP_PROXY;
 }
+#endif
```

Comment:
Nit: A /* corresponding #if */ would really help understand what is the corresponding #ifdef here.